products/amazon-linux.md

```diff
@@ -30,6 +30,10 @@ The original [Amazon Linux AMI][al1] is now in a [maintenance support phase][al1
 
 Security notifications for both versions are announced on the [Amazon Linux Security Center](https://alas.aws.amazon.com/). You can subscribe to [their RSS feed](https://alas.aws.amazon.com/alas.rss).
 
+## Amazon Linux 2022
+
+AWS announced a [Amazon Linux 2022](https://aws.amazon.com/linux/amazon-linux-2022/) as a successor to Amazon Linux 2, which uses Fedora as the upstream and is currently in preview. Starting with AL2022, a new major version of Amazon Linux releases every two years, includes minor quarterly releases, and comes with five years of long-term support. Every major version release will be supported under active development for 2 years, followed by a maintenance phase of 3 years. Security notifications are announced [here](https://alas.aws.amazon.com/alas2022.html) and come with a [RSS Feed](https://alas.aws.amazon.com/AL2022/alas.rss).
```

Comment:
isn't that `a` part is a typo ? i mean in `AWS announced a Amazon Linux 2022 as`